doc/installation.md

```diff
@@ -174,6 +178,10 @@ packages=com.navercorp
 
 - Setting up the agent via `-javaagent:path/to/scavenger-agent.jar`
 - Specify the config file via `-Dscavenger.configuration=path/to/scavenger.conf`
+- To change the default value of 10MB for the max message size of gRPC in agent, change the setting below.
```

Comment:
```suggestion
- To change the max message size of gRPC in agent, change the setting below.
```

Reply:
fixed it in 755bb3d91980ac24c9572a060398d1c60a1b5bd7